### ISSUE TITLE
[B13] Essentia particle streams not always visible

### ISSUE THREAD

**Issue by Geethebluesky** [Azanor/thaumcraft-beta#745]
"Not always" meaning it seems random when it shows up and when it doesn't, but it seems insivible more often than not. This is with a Thaumium smelter + single alembic + single Filling transfuser, going to a wall of jars.

In some cases, the stream becomes visible partway through a big essentia transfer: I saw it appear at the start of a 160+ Metallum transfer, and become invisible at some point while the smelter was finishing. 

At another time, I thought it wasn't transferring at all, so I stared at my jars to make sure they were filling anyway, and they were (just without a visible stream going from the transfuser to the jar itself). There doesn't seem to be a pattern to which essentia streams show up and which don't.

I don't know yet if this also happens with the infusion altar because I decided to keep infusion in general (and golems) for last.

Using Optifine D2 and Forge 2678 with     B:forgeLightPipelineEnabled=true

**Comment by Geethebluesky** [comment]
Ok I'm guessing this is by design now, since whenever the smelter got full of one type that couldn't drain for any reason, a stream appeared (probably because there was enough "pressure" for it to be worth drawing it). It's just that Essentia draining unit by unit isn't fast enough for this.

**Comment by Geethebluesky** [comment]
Okay well I should have waited until I did a few infusions :) The issue is present with the Altar as well, I just completed 2 Essentia Mirrors successfully, but only saw 1 out of 3 streams either time, and the ones I did see were different both times. Not seeing the streams made me worry that I had not set up the infusion correctly after all, or something I didn't see went wrong.

I also did more Transfusing with the smelter and a stream stopped right in the middle of sending Ordo to a jar.
